pkg/build/builder/common.go

```diff
@@ -20,7 +22,14 @@ import (
 // client facing libraries should not be using glog
 var glog = utilglog.ToFile(os.Stderr, 2)
 
-const OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
+const (
+	OriginalSourceURLAnnotationKey = "openshift.io/original-source-url"
```

Comment:
@csrwng it looks like you added this.. is anything using this other than the git server?  are we going to break anything meaningful if we change the annotation name?